sigs.yml

```
@@ -583,6 +583,20 @@
     gcLiaison:
     - name: Ted Young
       github: tedsuo
+  - name: eBPF Instrumentation
+    meeting: Wednesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1ZkmUT2EHKfgtLqrgx3WI8aBy2QNyZeTwSKXxe3DI6Pw
+    chat:
+      - type: slack
+        name: '#otel-ebpf-instrumentation'
+        id: C08P9L4FPKJ
+    invites: calendar-ebpf-instrumentation
+    sponsors: []
+    gcLiaison:
+      - name:
+        github:
```

```suggestion
      - name: Severin Neumann
        github: svrnm
```

I can serve as liaison, although I have to note that the meeting time conflicts with the weekly GC meeting, so I have to see how I will juggle attendence, or if I do checkins etc async.